package.json

```diff
@@ -20,11 +20,11 @@
   "author": "Kerim Satirli",
   "license": "Apache-2.0",
   "dependencies": {
```

Comment:
should everything be a prod runtime dependency?

Reply:
Unlikely - I'll clean this up in a future PR and make it better.

Thanks for pointing this out!